sktime/model_selection.py

```diff
@@ -225,3 +233,35 @@ def __init__(self, estimator, param_grid, scoring=None, fit_params=None,
             # using mean squared error as default for regressors
             elif isinstance(self.estimator, BaseRegressor):
                 self.scoring = make_scorer(mean_squared_error)
+
+class SKTimeCV(ABC):
+    def split(self, data):
+        """
+        Implements the splits to return the actual data instead of the 
+        """
+
+class PredefinedSplit:
```

Comment:
PredefinedSplit is already a class in sklearn, perhaps call it PresplitFilesCV or so to avoid ambiguity

Reply:
ok